modules/nf-core/calder2/meta.yml

```diff
@@ -0,0 +1,39 @@
+name: "calder2"
+description: Hierarchical Hi-C compartment computation
+keywords:
+  - calder2
```

Comment:
More keywords please! make the module searchable, e.g. `hi-c` etc. you can see more examples/suggestions in the green tags here: https://nf-co.re/modules